ts/components/dialog/ReactListModal.tsx

```diff
@@ -230,7 +252,20 @@ export const ReactListModal = (props: Props): ReactElement => {
     if (senders.length > 0 && (!reactionsMap[currentReact]?.senders || isEmpty(_senders))) {
       setSenders([]);
     }
-  }, [currentReact, me, reaction, msgProps?.sortedReacts, reactionsMap, senders]);
+
+    if (reactionsMap[currentReact]?.count && !isEqual(count, reactionsMap[currentReact]?.count)) {
```

Comment:
not sure why you need the isEqual here, those are just number right?

Reply:
Fixed + 1